app/assets/src/components/views/compare/SamplesHeatmapVis.jsx

```diff
@@ -126,8 +130,11 @@ class SamplesHeatmapVis extends React.Component {
 
   extractTaxonLabels() {
     return this.props.taxonIds.map(id => {
+      const name = this.props.taxonDetails[id].name;
       return {
-        label: this.props.taxonDetails[id].name,
+        // Moves missing to the top in "genus" sort.
+        // See MISSING_GENUS_ID in TaxonLineage.
+        label: id === -200 ? "* " + name : name,
```

Comment:
Aren't all negative genus a special case?

Reply:
Also, I think if treated specially these should be moved to the bottom.

Reply:
agree with Tiago that taxa with missing genus should be moved to the bottom.